dist/debian: drop dependencies on jdk-8

tchaikov opened the pull request:
this change mirrors https://github.com/scylladb/scylla-jmx/commit/f89670dfd80ed04860e1098f0fc6ef114793f31b,
since the oldest supported debian and its derivative are debian 10 and
ubuntu 18.04, both of which ship openjdk-11-jre-headless. and
oracle provides oracle-java11-set-default for them. so it's safe
to drop the optional dependencies on openjdk-8 and orcale-java8.

this would ensure that the scylla-jmx deb package's access to
jdk-11.

Fixes https://github.com/scylladb/scylla-jmx/issues/194

tchaikov commented:
@nyh and @denesb hi Nadav and Botond, could you help review this change?

avikivity commented:
Even centos 7 has java 11, so how about we align the dependencies everywhere?

tchaikov commented:
@avikivity we already updated rpm side in https://github.com/scylladb/scylla-jmx/commit/f89670dfd80ed04860e1098f0fc6ef114793f31b . This change is the missing puzzle.

avikivity commented:
Ok. But we have to merge the other change first, otherwise an upgrade to a version that only contains this patch will break.

tchaikov commented:
> Ok. But we have to merge the other change first, otherwise an upgrade to a version that only contains this patch will break.

are you referring #208 ?

avikivity commented:
yes, I see it's merged now.

avikivity commented:
The second patch was broken, I dropped it.

```
19:43:47  dpkg-buildpackage: error: syntax error in debian/control at line 12: duplicate field Depends found
```

tchaikov commented:
> The second patch was broken, I dropped it.
> 
> ```
> 19:43:47  dpkg-buildpackage: error: syntax error in debian/control at line 12: duplicate field Depends found
> ```

sorry. fixed and created as #210
